backend/lucene/src/main/java/org/hibernate/search/backend/lucene/scope/model/impl/LuceneScopeModel.java

```diff
@@ -148,4 +148,13 @@ public void checkNestedField(String absoluteFieldPath) {
 			throw log.unknownFieldForSearch( absoluteFieldPath, getIndexesEventContext() );
 		}
 	}
+
+	public boolean isNestedField(String absoluteFieldPath) {
```

Comment:
Why didn't you simply include this information in the object returned by `getSchemaNodeComponent`? Since we already call that method every time we need this information.